src/Microsoft.FeatureManagement/ConfigurationWrapper.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+using System;
+using System.Collections.Generic;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Primitives;
+
+namespace Microsoft.FeatureManagement
+{
+    /// <summary>
+    /// Wraps an instance of IConfiguration.
+    /// </summary>
+    class ConfigurationWrapper : IConfiguration
```

Comment:
Can you please help me understand what we use this wrapper class for?

Reply:
This Wrapper class allows us to update the "ref" when configurations are updated- regardless if the configuration updated it's reference or not. I believe that's it's only purpose- unless Jimmy had other reasons in mind.*

Reply:
Got it. Thanks. Would you mind adding some comments to clarify the necessity of this?

Reply:
Good idea, I'll do that